website/docs/registry/faq.mdx

```diff
@@ -0,0 +1,49 @@
+
+### How do I remove a version of my provider, module, or policy library that has already been published?
+
+If any users have already begun using a version of your provider, module, or policy library, removing the version from the Terraform Registry will break their Terraform configuration. They may be unable to work with their infrastructure until they can find a suitable replacement version of your integration. For this reason, **we recommend that you never remove previously published versions of providers, modules, or policy libraries**.
+
+If there is a bug in a certain version, it is usually better to publish another release, incrementing the patch version (i.e. from `1.2.0` to `1.2.1`), rather than to remove the bugged version.
+
+There may be situations where it is important that a version be removed from the Registry. Perhaps a version of a provider has a critical security vulnerability, or you may have extremely high confidence that no one is using a version of your module that doesn't work as you intended it to.
+
+In these situations, your options for removing previously published versions vary depending on the type of integration:
+
+- **Modules**: The owner of a module can use the Terraform Registry website to remove a specific version, all versions of a specific module provider, or remove the module entirely. [More details on this workflow can be found here](/registry/modules/publish#removing-published-content). or [here for policy libraries].
+- **Providers**: There is no manual process for removing a version. Reach out to us via email if it is critical that a version be deleted.
+- **Policy Libraries**: The owner of a policy library can completely remove it from the Registry, through the website. [More details on this workflow can be found here](/registry/policy-libraries/publishing#removing-a-policy-library).
+
+### Can I remove a version of a provider by deleting the release from GitHub?
+
+**We recommend you never remove content from a GitHub repository that has already been published to the Terraform Registry**.
+
+The Terraform Registry acts as a kind of index for Terraform-related content that is hosted on GitHub. When you make a new tag or a new release in a GitHub repository that holds your Terraform integration, the Registry automatically creates an index record for a new content. Think of that index record as simply a pointer to the files that exist in your GitHub repository.
+
+If you remove a file, or a tag, a release, or other content from GitHub, the Registry will not be made aware that you did so. The Registry may still host index records pointing to the now-removed files or content. As a result, someone using your provider or module may see their Terraform configuration break, since through the Registry it expects to find files on GitHub which are no longer available.
```

Comment:
Would be great to include the recommended path alongside this. Maybe something like:

"Instead, if you discover a flaw in a published version of your module or provider, we recommend publishing a new version with the flaw fixed."

Reply:
Ah, I see you say this elsewhere in the FAQ - maybe we make it a separate question and then link to it from these other questions?